vision_agent_tools/shared_types.py

```diff
@@ -1,10 +1,21 @@
+from typing import Annotated, Literal, TypeVar
+
 from pydantic import BaseModel
+import numpy as np
+import numpy.typing as npt
 
 
 class BaseTool:
     pass
 
 
+DType = TypeVar("DType", bound=np.generic)
+
+VideoNumpy = Annotated[npt.NDArray[DType], Literal["N", "N", "N", 3]]
```

Comment:
@dillonalaird I think this value is already defined inside the file `vision_agent_tools/types.py` We should probably merge together the `shared_types.py` file that you are moving out of the `tools` folder.

Reply:
I merged them. I moved it to this file.

Reply:
sorry, I thought I deleted the other file but I did not. I will delete the file, but I migrated all the imports to use this file.

Reply:
I just pushed the changes